package.json

```diff
@@ -208,6 +208,7 @@
     "json-stable-stringify": "^1.0.1",
     "json-stringify-pretty-compact": "1.2.0",
     "json-stringify-safe": "5.0.1",
+    "kea": "^2.0.1",
```

Comment:
IIRC should go into `x-pack/package.json`, not the main `package.json` itself, since we're a plugin separate from core

Reply:
Excellent. Thanks!